its/src/test/java/com/sonar/it/vbnet/Tests.java

```diff
@@ -77,6 +77,10 @@ public static void deleteLocalCache() {
     TestUtils.deleteLocalCache();
   }
 
+  static BuildResult analyzeProject(TemporaryFolder temp, String projectName) throws IOException {
```

Comment:
To keep it aligned with CSharp (we don't need all those overloads, but the existing ones should match)
```suggestion
  static BuildResult analyzeProject(TemporaryFolder temp, String projectDir) throws IOException {
```